### ISSUE TITLE
Possible broken dependecy of dagit (requiring dagster 1.1.12, which is not published)

### ISSUE THREAD

**Issue by sergioisidoro** [dagster-io/dagster#11816]
### Dagster version

1.1.11

### What's the issue?

While installing dagit I get an error that complains of a non existing dependency of dagster >1.1.12, which I see does not yet exist (pypi still on 1.1.11 on the latest). 

```
Because no versions of dagit match >1.1.12,<2.0.0
 and dagit (1.1.12) depends on dagster (1.1.12), dagit (>=1.1.12,<2.0.0) requires dagster (1.1.12).
So, because no versions of dagster match 1.1.12
 and <my-project> depends on dagit (^1.1.12), version solving failed.
 ```
I see that 1.1.11 of dagster was released just 15h ago, and dagit 1.1.12 was just released today. I guess this was a mishap or a forgotten version bump? 

### How to reproduce?

`pip install dagster dagit` on a clean machine

**Comment by sergioisidoro** [comment]
I see the tag 1.1.12 exists (but not a release), and it seems it was not built and pushed to pypi, unlike dagit.
https://github.com/dagster-io/dagster/releases/tag/1.1.12

**Comment by elementljarredc** [comment]
thanks!

**Comment by alangenfeld** [comment]
this was a weird state that pypi got in to (filed here https://github.com/pypi/warehouse/issues/12876)

appears to have resolved with the 1.1.13 release
